extensions/ql-vscode/src/remote-queries/view/RepositoriesSearch.tsx

```diff
@@ -14,7 +14,8 @@ const RepositoriesSearch = ({ filterValue, setFilterValue }: RepositoriesSearchP
       block
       sx={{
         backgroundColor: 'var(--vscode-editor-background);',
-        color: 'var(--vscode-editor-foreground);'
+        color: 'var(--vscode-editor-foreground);',
+        width: 'calc(100% - 14px)',
```

Comment:
Why is this needed? If I turn it off things look the same in my view. Is it trying to add a padding of some sort?

Reply:
Yes. The width of the search bar needs to be exactly the same as the width of the analysis results section. I was trying to find a way to get this to work using padding or margin, but because we are using primer, we can't attach css to the elements that would be affected by padding or margin elements. This is the only way I could figure out how to get it working.